documentation/source/users/rmg/running.rst

```diff
@@ -4,33 +4,107 @@
 
 Basic run::
 
 	python rmg.py input.py
 
-Run by restarting from a seed mechanism ::
+.. _inputflags:
+
+Input flags
+-----------
+
+The options for input flags can be found in ``/RMG-Py/rmgpy/util.py``. Running ::
+
+ 	python rmg.py -h
+
+at the command line will print the documentation from ``util.py``, which is reproduced below for convenience::
+
+	usage: rmg.py [-h] [-q | -v | -d] [-o DIR] [-r path/to/seed/] [-p] [-P]
+              [-t DD:HH:MM:SS] [-i MAXITER] [-n MAXPROC] [-k]
+              FILE
+
+	Reaction Mechanism Generator (RMG) is an automatic chemical reaction mechanism
+	generator that constructs kinetic models composed of elementary chemical
+	reaction steps using a general understanding of how molecules react.
+
+	positional arguments:
+	  FILE                  a file describing the job to execute
+
+	optional arguments:
+	  -h, --help            show this help message and exit
+	  -q, --quiet           only print warnings and errors
+	  -v, --verbose         print more verbose output
+	  -d, --debug           print debug information
+	  -o DIR, --output-directory DIR
+	                        use DIR as output directory
+	  -r path/to/seed/, --restart path/to/seed/
+	                        restart RMG from a seed
+	  -p, --profile         run under cProfile to gather profiling statistics, and
+	                        postprocess them if job completes
+	  -P, --postprocess     postprocess profiling statistics from previous
+	                        [failed] run; does not run the simulation
+	  -t DD:HH:MM:SS, --walltime DD:HH:MM:SS
+	                        set the maximum execution time
+	  -i MAXITER, --maxiter MAXITER
+	                        set the maximum number of RMG iterations
+	  -n MAXPROC, --maxproc MAXPROC
+	                        max number of processes used during reaction
+	                        generation
+	  -k, --kineticsdatastore
+	                        output a folder, kinetics_database, that contains a
+	                        .txt file for each reaction family listing the
+	                        source(s) for each entry
+
+Some representative example usages are shown below.
+
+Run by restarting from a seed mechanism::
 
     python rmg.py input.py -r path/to/seed/
 
-Run with CPU profiling::
+Run with CPU time profiling::
 
     python rmg.py input.py -p
 
 Run with multiprocessing for reaction generation and QMTP::
 
     python rmg.py -n <Max number of processes allowed> input.py 
 
+Run with setting a limit on the maximum execution time::
+
+	python rmg.py input.py -t <DD:HH:MM:SS>
+
+Run with setting a limit on the maximum number of iterations::
+
+	python rmg.py input.py -i <Max number of desired iterations>
+
 We recommend you make a job-specific directory for each RMG simulation. Some jobs can take quite a while to complete, so we also recommend using a job scheduler if working in a linux environment. 
```

Comment:
This final sentence probably belongs better in the _Running a Job_ section than the _Input Flags_ section, as it is not to do with input flags.

Reply:
Good point. Thanks!